### ISSUE TITLE
Node group roles for deleted nodes

### ISSUE THREAD

**Issue by Thiblef** [open-orchestra/open-orchestra-cms-bundle#1739]
## Informations

**Orchestra Version** : 
1.1
**Issue Type**: 

- [x] Bug
- [ ] Feature request
- [ ] Question

## Description

Node modelGroupRoles for deleted nodes (in trashcan) are not created on group creation.

### Actual behavior
It produces a 500 error when you want to update a parent node model group role:
`NodeGroupRoleNotFoundException` thrown from `UpdateNodeGroupRoleListener` line 55 : 

```
if ($nodeGroupRole === null) {
     throw new NodeGroupRoleNotFoundException($role, $node->getNodeId(), $group->getName());
} else if (ModelGroupRoleInterface::ACCESS_INHERIT === $nodeGroupRole->getAccessType()) {
     $nodeGroupRole->setGranted($document->isGranted());
}

```
### Expected behavior
Maybe create the modelGroupRoles for deleted nodes in `NodeGroupRoleForGroupSubscriber::prePersist()`.
### Steps to reproduce the problem
1. Create a node
2. Delete the node
3. Create a group on the node site
4. Try to update a parent right in node role list of the group
5. Error 500 : `The role 'ROLE_....' of node 'nodeId' was not found in group 'groupId'`

**Comment by itkg-nanne** [comment]
in roadmap

**Comment by Thiblef** [comment]
For info:
For the same reason, same exception is thrown when you want to move a node which contains a deleted node.
From `UpdateNodeGroupRoleMoveNodeSubscriber::updateNodeGroupRoleTree()` line 77.

**Comment by itkg-nanne** [comment]
fixed in 1.2 : https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1753
fixed in 1.1 : https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1754
